src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        switch (new Random().nextInt(Shapes.values().length) + 1) {
+            case 1: return new Circle(
+                    new Random().nextDouble(100),
+                    ColorSupplier.getRandomColor());
+            case 2: return new Square(
+                    new Random().nextDouble(100),
+                    ColorSupplier.getRandomColor());
+            case 3: return new Rectangle(
```

Comment:
There is no reason to write return in all cases, it is possible to create a variable on method level and populate it